packages/mobile/src/components/core/UserGeneratedText.tsx

```diff
@@ -177,32 +177,34 @@ export const UserGeneratedText = (props: UserGeneratedTextProps) => {
     useState<LayoutRectangle>()
 
   useEffect(() => {
-    let layouts = {}
-    const linkKeys = Object.keys(links)
+    if (allowPointerEventsToPassThrough) {
+      let layouts = {}
+      const linkKeys = Object.keys(links)
 
-    // Measure the layout of each link
-    linkKeys.forEach((key) => {
-      const linkRef = linkRefs[key]
-      if (linkRef) {
-        // Need to use `measureInWindow` instead of `onLayout` or `measure` because
-        // android doesn't return the correct layout for nested text elements
-        linkRef.measureInWindow((x, y, width, height) => {
-          layouts = { ...layouts, [key]: { x, y, width, height } }
+      // Measure the layout of each link
```

Comment:
ew what happened with formatting here

Reply:
I just wrapped the whole thing in a conditional so it indented. I could do an early return, but I think this is fine

Reply:
oh I see, gotcha